src/flows/context/editor.tsx

```diff
@@ -7,23 +7,19 @@ import React, {
 } from 'react'
 import {EditorType} from 'src/types'
 import {PipeContext} from 'src/flows/context/pipe'
-
-export enum InjectionType {
-  OnOwnLine = 'onOwnLine',
-  SameLine = 'sameLine',
-}
-
-interface InjectionPosition {
-  row: number
-  column: number
-  shouldStartWithNewLine: boolean
-  shouldEndInNewLine: boolean
-}
+import {
+  InjectionType,
+  InjectionPosition,
+  calcInjectionPosition,
+} from 'src/shared/utils/fluxFunctions'
+export {InjectionType, InjectionPosition} from 'src/shared/utils/fluxFunctions'
```

Comment:
Note that making a generalized editor.context will be also required for part of the QE flux builder. (Because all the schema injections and function injections will use the same logic). So this follow-up PR will definitely be occurring. 😸